models/stg_linkedin_ads__ad_analytics_by_campaign.sql

```diff
@@ -0,0 +1,47 @@
+
+with base as (
+
+    select * 
+    from {{ ref('stg_linkedin_ads__ad_analytics_by_campaign_tmp') }}
+),
+
+macro as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_linkedin_ads__ad_analytics_by_campaign_tmp')),
+                staging_columns=get_ad_analytics_by_campaign_columns()
+            )
+        }}
+    from base
+),
+
+fields as (
+    
+    select 
+        cast(day as {{ dbt_utils.type_timestamp() }}) as date_day,
```

Comment:
Let's discuss as a larger team, but I am curious why some platforms have this field as a truncated day and others have a timestamp. I want to determine if it would be best practice to truncate to a single day in the staging models.

Reply:
yeah this package has a lot of weird vestiges -- i think we should truncate